io.openems.edge.meter.api/src/io/openems/edge/meter/api/ElectricityMeter.java

```diff
@@ -2,136 +2,211 @@
 
 import java.util.function.Consumer;
 
+import org.osgi.annotation.versioning.ProviderType;
+
 import io.openems.common.channel.AccessMode;
 import io.openems.common.channel.PersistencePriority;
 import io.openems.common.channel.Unit;
 import io.openems.common.types.OpenemsType;
+import io.openems.common.utils.IntUtils;
+import io.openems.common.utils.IntUtils.Round;
+import io.openems.edge.common.channel.Channel;
 import io.openems.edge.common.channel.Doc;
+import io.openems.edge.common.channel.IntegerDoc;
 import io.openems.edge.common.channel.IntegerReadChannel;
 import io.openems.edge.common.channel.LongReadChannel;
 import io.openems.edge.common.channel.value.Value;
+import io.openems.edge.common.component.OpenemsComponent;
 import io.openems.edge.common.modbusslave.ModbusSlaveNatureTable;
 import io.openems.edge.common.modbusslave.ModbusType;
 import io.openems.edge.common.type.TypeUtils;
 
 /**
- * Represents an Asymmetric Meter.
- *
+ * Represents an electricity Meter.
+ * 
+ * <p>
+ * This is the parent of everything that measures electricity flow. Consider
+ * using:
  * <ul>
- * <li>Negative ActivePowerL1/L2/L3 and ConsumptionActivePowerL1/L2/L3 represent
- * Consumption, i.e. power that is 'leaving the system', e.g. feed-to-grid
- * <li>Positive ActivePowerL1/L2/L3 and ProductionActivePowerL1/L2/L3 represent
- * Production, i.e. power that is 'entering the system', e.g. buy-from-grid
+ * <li>{@link SymmetricMeter}: if L1, L2 and L3 are always equal
```

Comment:
I assume the SymmetricMeter will be removed completely, right?
Maybe it makes sense to change the description for symmetric systems, to use the total amount divided by three